syntax="proto2";
package test;

option cc_generic_services = true;

message TestRequest {
  optional string name = 1;
  optional int32 num = 2;
};
message TestResponse {
  optional int32 err = 1;
  optional string message = 2;
};

message HttpRequest {

};
message HttpResponse {

};

service HttpService {
  rpc Echo(HttpRequest) returns (HttpResponse);
  rpc EchoProtobuf(HttpRequest) returns (HttpResponse);
};


service QueueService {
  rpc start(HttpRequest) returns (HttpResponse);
  rpc stop(HttpRequest) returns (HttpResponse);
  rpc getstats(HttpRequest) returns (HttpResponse);
};
